src/pages/governance/_components/VotingResult.tsx

```diff
@@ -386,56 +346,75 @@ export function VotingResult({
 }
 
 function Progress({
-  value,
+  yesValue,
+  noValue,
   containerClass,
-  contentClass,
+  approvalThreshold,
 }: {
-  value: number;
+  yesValue: number;
+  noValue: number;
+  approvalThreshold: number;
   containerClass: string;
-  contentClass: string;
 }) {
   return (
-    <div
-      className={classNames(
-        "h-3 rounded-[54px] w-full bg-gray-200 dark:bg-dark-gray-200 relative overflow-hidden",
-        containerClass
-      )}
-    >
+    <div className="h-6 relative">
+      <div
+        style={{ width: `${approvalThreshold}%` }}
+        className="absolute left-0 -top-1.5 h-6 border-r border-gray-200 dark:border-dark-gray-200"
+      />
       <div
-        style={{ width: `${value}%` }}
         className={classNames(
-          "absolute top-0 left-0 h-3 bg-primary-500 dark:bg-dark-primary-500",
-          contentClass
+          "h-3 rounded-[54px] w-full bg-gray-200 dark:bg-dark-gray-200 relative overflow-hidden",
+          containerClass
         )}
-      />
+      >
+        <div
+          style={{ width: `${yesValue}%` }}
+          className={classNames(
+            "absolute top-0 left-0 h-3 bg-primary-500 dark:bg-dark-primary-500",
+            "bg-green-600 dark:bg-[#21E529]"
+          )}
+        />
+        <div
+          style={{ width: `${noValue}%` }}
+          className={classNames(
+            "absolute top-0 right-0 h-3 bg-primary-500 dark:bg-dark-primary-500",
+            "bg-red-600 dark:bg-[#FF483D]"
+          )}
+        />
+      </div>
     </div>
   );
 }
 
 function LabelWithInfoTooltipAndChecks({
   labelTitle,
   value,
-  suffix,
+  baseValue,
   comparatorValue,
+  proposalStatus,
+  suffix,
   toolTipDesc,
   decimalPlace,
 }: {
   labelTitle: string;
   value: BigNumber;
-  suffix?: string;
+  baseValue?: BigNumber;
   comparatorValue?: BigNumber;
```

Comment:
- from what i see, comparatorValue is the same as value, can we reuse it and dont have to reintroduce another repeated variable?
- to me `baseValue` is more like the `comparatorValue`, assuming `value` is the one we are going to display

Reply:
updated variables for component